evap/staff/views.py

```diff
@@ -2262,61 +2274,59 @@ def user_merge(request, main_user_id, other_user_id):
 
-    if template.name == EmailTemplate.STUDENT_REMINDER:
-        available_variables += ["first_due_in_days", "due_evaluations"]
-    elif template.name in [
-        EmailTemplate.EDITOR_REVIEW_NOTICE,
-        EmailTemplate.EDITOR_REVIEW_REMINDER,
-        EmailTemplate.PUBLISHING_NOTICE_CONTRIBUTOR,
-        EmailTemplate.PUBLISHING_NOTICE_PARTICIPANT,
-    ]:
-        available_variables += ["evaluations"]
-    elif template.name == EmailTemplate.TEXT_ANSWER_REVIEW_REMINDER:
-        available_variables += ["evaluation_url_tuples"]
-    elif template.name == EmailTemplate.EVALUATION_STARTED:
-        available_variables += ["evaluations", "due_evaluations"]
-    elif template.name == EmailTemplate.DIRECT_DELEGATION:
-        available_variables += ["evaluation", "delegate_user"]
-
-    available_variables = ["{{ " + variable + " }}" for variable in available_variables]
-    available_variables.sort()
+        available_variables = [
+            "contact_email",
+            "page_url",
+            "login_url",  # only if they need it
+            "user",
+        ]
 
-    return render(
-        request,
-        "staff_template_form.html",
-        {"form": form, "template": template, "available_variables": available_variables},
-    )
+        if template.name == EmailTemplate.STUDENT_REMINDER:
+            available_variables += ["first_due_in_days", "due_evaluations"]
+        elif template.name in [
+            EmailTemplate.EDITOR_REVIEW_NOTICE,
+            EmailTemplate.EDITOR_REVIEW_REMINDER,
+            EmailTemplate.PUBLISHING_NOTICE_CONTRIBUTOR,
+            EmailTemplate.PUBLISHING_NOTICE_PARTICIPANT,
+        ]:
+            available_variables += ["evaluations"]
+        elif template.name == EmailTemplate.TEXT_ANSWER_REVIEW_REMINDER:
+            available_variables += ["evaluation_url_tuples"]
+        elif template.name == EmailTemplate.EVALUATION_STARTED:
+            available_variables += ["evaluations", "due_evaluations"]
+        elif template.name == EmailTemplate.DIRECT_DELEGATION:
+            available_variables += ["evaluation", "delegate_user"]
 
+        available_variables = ["{{ " + variable + " }}" for variable in available_variables]
+        available_variables.sort()
 
-@manager_required
-def faq_index(request):
-    sections = FaqSection.objects.all()
+        context["available_variables"] = available_variables
 
-    SectionFormset = modelformset_factory(FaqSection, form=FaqSectionForm, can_delete=True, extra=1)
-    formset = SectionFormset(request.POST or None, queryset=sections)
+        return context
 
-    if formset.is_valid():
-        formset.save()
-        messages.success(request, _("Successfully updated the FAQ sections."))
-        return redirect("staff:faq_index")
 
-    return render(request, "staff_faq_index.html", {"formset": formset, "sections": sections})
+@manager_required
+class FaqIndexView(SuccessMessageMixin, SaveValidFormMixin, FormsetView):
```

Comment:
codecov says untested on the class definition line? This got to be wrong, right?